### PR TITLE
Feature/filacms 7 create authors table

### PR DESCRIPTION
**Pull request by jeremy-portable**

### CODE REVIEW

File: database/migrations/2024_02_09_300000_create_authors_table.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('authors', function (Blueprint $table) {
+            $table->id();
+            $table->string('first_name');
+            $table->string('last_name')->nullable();
+            $table->unsignedTinyInteger('is_individual')->default(1);
```
Comment (kyoungportable) [inline database/migrations/2024_02_09_300000_create_authors_table.php:18]: I would use `->boolean('is_individual')->default(false)` here I think

File: src/Models/Author.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        'first_name',
+        'last_name',
+        'is_individual',
+    ];
+
+    protected $casts = [
+        'is_individual' => 'boolean',
+    ];
+
+    public function getDisplayNameAttribute()
```
Comment (kyoungportable) [inline src/Models/Author.php:19]: Can you please use the Laravel attribute object (new way of doing attributes)? https://laravel.com/docs/10.x/eloquent-mutators#accessors-and-mutators

**Review by kyoungportable — CHANGES_REQUESTED** [review]
Please see comments :)

File: database/migrations/2024_02_09_300000_create_authors_table.php (hunk 1 above)
Comment (jeremy-portable) [reply database/migrations/2024_02_09_300000_create_authors_table.php:18]: Yep we could do that too, though boolean is a synonym of tinyint in the background so I got used to that. But I'll implement boolean.

File: src/Models/Author.php (hunk 2 above)
Comment (jeremy-portable) [reply src/Models/Author.php:19]: Got it

**Review by kyoungportable — APPROVED** [review]

**Pull request merged**